t/cobrand/kingston.t

```diff
@@ -0,0 +1,894 @@
+                                '@type' => "MandateDTO",
+                                payerId => 329,
+                                profileId => 200,
+                                created => "21/02/2021",
+                                lastUpdated => "21/02/2021",
+                                accountName => "A Payer",
+                                accountNumber => 123,
+                                reference => "GGW854325",
+                                sortCode => "12345",
+                                status => "CANCELLED",
+                            } } ] } ] },
+                            { values => [ { resultValues => [ { value => {   # cancel nothing in echo
+                                '@type' => "MandateDTO",
+                                payerId => 103,
+                                profileId => 200,
+                                created => "21/02/2021",
+                                lastUpdated => "26/02/2021",
+                                accountName => "A Payer",
+                                accountNumber => 123,
+                                reference => "GGW954324",
+                                sortCode => "12345",
+                                status => "CANCELLED",
+                            } } ] } ] },
+                            { values => [ { resultValues => [ { value => {   # cancel no extended data
+                                '@type' => "MandateDTO",
+                                payerId => 24,
+                                profileId => 200,
+                                created => "26/02/2021",
+                                lastUpdated => "26/02/2021",
+                                accountName => "A Payer",
+                                accountNumber => 123,
+                                reference => "GGW6654326",
+                                sortCode => "12345",
+                                status => "CANCELLED",
+                            } } ] } ] },
+                        ]
+                };
+            }
+        });
+
+        my $contact = $mech->create_contact_ok(body => $body, category => 'Garden Subscription', email => 'garden@example.com');
+        $contact->set_extra_fields(
+                { name => 'uprn', required => 1, automated => 'hidden_field' },
+                { name => 'property_id', required => 1, automated => 'hidden_field' },
+                { name => 'service_id', required => 0, automated => 'hidden_field' },
+                { name => 'Subscription_Type', required => 1, automated => 'hidden_field' },
+                { name => 'Subscription_Details_Quantity', required => 1, automated => 'hidden_field' },
+                { name => 'Subscription_Details_Container_Type', required => 1, automated => 'hidden_field' },
+                { name => 'Container_Instruction_Quantity', required => 1, automated => 'hidden_field' },
```

Comment:
These are different too, Bin_Delivery_Detail with different endings (presumably this bit doesn't matter for this though).